Fix formatted Kanban card text including HTML tags (#19112)
directus/directus: .changeset/plenty-dragons-drop.md

---
'@directus/app': patch
---

> Fixed displaying of formatted values in Kanban cards

Reviewer: ```suggestion
Fixed displaying of formatted values used as card text in Kanban layout
```

> Fixed displaying of formatted values used as card text in Kanban layout

Reviewer: ```suggestion
Fixed display of formatted values used as card text in Kanban layout
```

---
'@directus/app': patch
---

> The Card Text of the Kanban board layout items have been changed to use the given field's formatted value when the Card
> Text field is a WYSIWYG

Reviewer: ```suggestion
Fixed displaying of formatted values in Kanban cards
```
